packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -734,6 +743,27 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		return typeof this.rowClassNameFn === "function" ? this.rowClassNameFn(row, index) : "";
 	}
 
+	/**
+	 * Handles if a row is clicked. If there are listeners on the rowClick event of the table these should handle the event.
+	 * If there are no listeners we fall back to the default behaviour, which is (de)selecting the row (if multiSelect is enabled)
+	 * @param row - The data object passed to the row
+	 */
+	public onRowClick(row: any): void {
+		// If there is a observer, emit an event
+		if (this.rowClick.observers.length > 0) {
+			this.rowClick.emit(row);
+			return;
+		}
+
+		// If multi-select is enabled, select the row
+		if (this.isMultiSelectEnabled) {
```

Comment:
Previously, if the developer set an action on the row, we didn't select the row. It was the job of the dev if he wanted to select it... I think maybe it should stay as this, otherwise it will become really complex to understand what's happening when we click 😕 

What do you think about this ? 
@christophercr @dsebastien ? 😊

Reply:
I did do this, but I terminated the function if an event observer is found.
I could also do this with `else if` if it is preferred.

Reply:
Actually, to be honest, I didn't see the `return`s :-p
Anyways, I prefer to have `if`, `else if` than `return`. I find it more readable :blush: